event-handler/Dockerfile

```diff
@@ -5,18 +5,21 @@ FROM golang:${GO_VERSION} as builder
 
 WORKDIR /workspace
 
+# Copy .git directory needed for go build
+COPY .git .git
```

Comment:
Currently, the event handler passes the following build flags during compilation:

https://github.com/gravitational/teleport-plugins/blob/0d4c060b4523466951df027dc6c8be300a83d1a7/event-handler/Makefile#L14

Whereas, the access plugin `Makefiles` don't require it:

https://github.com/gravitational/teleport-plugins/blob/0d4c060b4523466951df027dc6c8be300a83d1a7/access/email/Makefile#L7-L8

The `event-handler` plugin uses the `Sha` in it's `version.go` file:
https://github.com/gravitational/teleport-plugins/blob/0d4c060b4523466951df027dc6c8be300a83d1a7/event-handler/version.go#L19-L25

## Alternatives
Instead of copying `./.git` into the intermediate build image, I could instead pass it as a build arg and then update the corresponding make invocations as appropriate.

Reply:
Filed https://github.com/gravitational/teleport-plugins/issues/453 about this.

I'm fine with copying `.git` in, I'd just like it to be consistent.